<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit_inputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1221</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_cipher">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Text to cipher:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_key">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_key">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>1221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cipher">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Cipher</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_ciphered">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>501</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_input0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_input1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_input2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_input3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_input4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_keystream0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_keystream1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_keystream4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_keystream2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_keystream3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_result0">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_result1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_result4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>490</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_result2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>410</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_result3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_xor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>XOR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_textbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_keystreambox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Keystream</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_res">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_final">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Result in Hex:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decipher">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Decipher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_decipher">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Text to decipher:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_hexa">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hexa</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_string">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>string</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_inputmode">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Mode: </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_inputTextHex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>1221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_inputHex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Text in Hex:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_finalPlain">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>300</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Result in clear text:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_cipheredPlain">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>330</y>
      <width>431</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Cipher</string>
    </property>
    <addaction name="actionRC4"/>
    <addaction name="actionAES"/>
    <addaction name="actionRSA"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuMode_2">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionCipher"/>
    <addaction name="actionDecipher"/>
   </widget>
   <addaction name="menuMode_2"/>
   <addaction name="menuMode"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRC4">
   <property name="text">
    <string>RC4</string>
   </property>
  </action>
  <action name="actionAES">
   <property name="text">
    <string>AES</string>
   </property>
  </action>
  <action name="actionRSA">
   <property name="text">
    <string>RSA</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCipher">
   <property name="text">
    <string>Cipher</string>
   </property>
  </action>
  <action name="actionDecipher">
   <property name="text">
    <string>Decipher</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
